s3_helper.py

```diff
@@ -50,14 +50,14 @@ def _create_aws_session(self):
         except botocore.exceptions.ProfileNotFound:
             self.print_func('Please supply a valid AWS profile name.')
             exit()
-        except:
+        except Exception as e:
             self.print_func(traceback.format_exc())
-            self.print_func('Exiting. Unable to establish AWS session with the following profile name: {}'.format(self.aws_profile))
+            self.print_func('Exiting. Unable to establish AWS session with the following profile name: {}'.format(self.aws_profile)) +str(e)
```

Comment:
Since we are already passing the exception traceback to `self.print_func` in line 54, we don't need to print the exception again in line 55, so let's revert that change.

As it is right now, the change in line 55 would also introduce a bug given that it's outside of the `self.print_func`.